### ISSUE TITLE
how to make "uri_ignore_words_list" work ?

### ISSUE THREAD

**Issue by chipitsine** [codespell-project/actions-codespell#55]
we use codespell in haproxy, I tried to ignore uri:

https://github.com/chipitsine/haproxy/commit/5e2fd695debdb8d9bee853cd98d0121c5942f8cb

uri are still detected:

https://github.com/chipitsine/haproxy/actions/runs/3670466510/jobs/6205020701

particularly

![image](https://user-images.githubusercontent.com/2217296/206924592-57cf0db8-9b33-4551-ab21-e3625eacf12c.png)

text is

```
    >>> Aug  9 20:30:46 localhost \
          haproxy[2022]: 127.0.0.1:34020 [09/Aug/2004:20:30:46] proxy-out \
          proxy-out/cache1 0/0/0/182/+182 200 +279 - - ---- 0/0/0/0/0 0/0 \
          {w.ods.org||} {Formilux/0.1.8|3495|||} \
          "GET http://trafic.1wt.eu/ HTTP/1.1"
```

**Comment by peternewman** [comment]
Agreed this seems to be a bug as the wildcard isn't working as expected. Specifying it manually works as you'll see here:
https://github.com/peternewman/haproxy/actions/runs/3671097944/jobs/6206070167

But you'll need to ignore the word in the GitHub workflow file then.

I believe the underlying codespell wildcard works, so this is probably something to do with how we're handling it in the Action. If you want to switch to using a config file, I suspect that will fix it too:
https://github.com/codespell-project/codespell#using-a-config-file

**Comment by shipitsin-ilia** [comment]
it is not documented how to use config with actions, is that possible?

**Comment by chipitsine** [comment]
any update on this ?
